### ISSUE TITLE
Support loading plugins without FS access

### ISSUE THREAD

**Issue by strogonoff** [davideicardi/live-plugin-manager#52]
Use case: install plugins in Electron’s window without `nodeIntegration`, and hence without filesystem access.

This use case could be addressed by allowing to install in-memory.

Which, in turn, could be supported by allowing to pass a custom conforming `fs` implementation to PluginManager (there’re API-compatible implementations of FS like LightningFS, for example).

**Comment by davideicardi** [comment]
Yes, good idea. It should be not too difficult to implement. As usual any help is appreciated ;-)
